### ISSUE TITLE
how to launch the tensorflow-model-server-neuron gRPC server at default port 8500 in the background successfully?

### ISSUE THREAD

**Issue by karayang** [aws/aws-neuron-sdk#151]
I wonder how to launch the tensorflow-model-server-neuron gRPC server at default port 8500 in the background correctly.
Here is what I did
1) Launching gRPC server: 
tensorflow_model_server_neuron --model_base_path=$(pwd)/ssd300_tf_neuron &
2) Then in the same terminal, I evaluate the Neuron-compatible TensorFlow SavedModel for both accuracy and performance using:
python ssd300_evaluation_client.py --val2017=./val2017 --instances_val2017_json=./annotations/instances_val2017.json
Problem:
The current throughput is always  0.4 so I assume it didn't connect to the gRPC server successfully.

Can anyone explain how to do these 2 steps correctly?
1) Launch the tensorflow-model-server-neuron gRPC server at default port 8500 in the background.
tensorflow_model_server_neuron --model_base_path=$(pwd)/ssd300_tf_neuron &

2)In client, evaluate the Neuron-compatible TensorFlow SavedModel for both accuracy and performance. Note that this client by default assumes a tensorflow-model-server-neuron listening at localhost:8500. On inf1.xlarge, the expected throughput is 83 images/second once the server is fully warmed up, and the expected mean average precision (mAP) is 0.253.
python ssd300_evaluation_client.py --val2017=./val2017 --instances_val2017_json=./annotations/instances_val2017.json

Thanks!

**Comment by awsrjh** [comment]
Hi there! Let me do some checking and get back to you -- thanks.

**Comment by happycoding621** [comment]
Any feedback? I also work for aws. Can I contact you?

**Comment by awsrjh** [comment]
Hi -- absolutely -  I have been working to solve this: I believe it is the instructions in the ssd300 example that have an incorrect installation based on some recent changes we made. I am working to verify that. 

Please feel free to reach me at rjheaton@amazon.com

**Comment by awsrjh** [comment]
Hi - So we think this is the solution: the current dependencies actually end up overwriting the tensorflow-neuron needed for this, and so it needs to be reinstalled.  (We will fix this shortly in the instructions).

After executing the steps in "Step 1 Install dependencies" in the setup guide here: https://github.com/aws/aws-neuron-sdk/tree/master/src/examples/tensorflow/ssd300_demo#compile-open-source-pytorch-ssd300-model-and-checkpoint-into-neuron-compatible-ssd300-tensorflow-savedmodel

please execute: 

pip install tensorflow-neuron --force --no-deps --extra-index-url=https://pip.repos.neuron.amazonaws.com

I just ran it and after doing this extra reinstall, I got these results on an inf1.2xlarge (snippet of the output):

pid 7130: current throughput 116.0, latency p50=0.034 p90=0.037
evaluating image 4600/4952
evaluating image 4700/4952
evaluating image 4800/4952
evaluating image 4900/4952
loading annotations into memory...
Done (t=0.56s)
creating index...
index created!
Loading and preparing results...
Converting ndarray to lists...
(404257, 7)
0/404257
DONE (t=2.35s)
creating index...
index created!
Running per image evaluation...
Evaluate annotation type *bbox*
DONE (t=49.76s).
Accumulating evaluation results...
DONE (t=5.12s).
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.253

**Comment by happycoding621** [comment]
Thanks!
